image_server/server_setup.md

```diff
@@ -78,7 +78,6 @@ If setting up in a browser, go to the CloudFormation service in the AWS console
 
 
 2. Specify parameters - some hints are:
```

Comment:
```suggestion
2. Specify parameters - some hints are:

   * Bucket - name of S3 bucket (not including `s3://`) where source rasters are stored, generally `asf-event-data` for disasters-related data
```

Reply:
Ah, I see your reply. The bucket variable is just in the config file now. Thanks!